google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerException.java

```diff
@@ -34,8 +34,7 @@
       boolean retryable,
       @Nullable String message,
       @Nullable Throwable cause) {
-    super(code.getCode(), message, null /* reason */, false /* idempotent */, cause);
-    this.retryable = retryable;
+    super(message, cause, code.getCode(), retryable);
```

Comment:
It's the SpannerException that has an odd order - the order of the base exception is the "normal" one (used in multiple places). I didn't want to mess with the SpannerException for now.